llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -1108,27 +1136,43 @@ void AsmPrinter::emitBBAddrMapSection(const MachineFunction &MF) {
     // always be computed from their offsets.
     emitLabelDifferenceAsULEB128(MBB.getEndSymbol(), MBBSymbol);
     OutStreamer->emitULEB128IntValue(getBBAddrMapMetadata(MBB));
-    // Emit the index of the corresponding LLVMIR basic block.
-    size_t BBIdx = 0;
-    bool found = false;
-    const BasicBlock *FindBB = MBB.getBasicBlock();
-    if (FindBB == nullptr) {
-      found = true;
-      BBIdx = NO_BB;
-    } else {
+    // Find BBs corresponding with this MBB.
```

Comment:
What does "corresponding" mean here?

Reply:
It's the BBs that correspond with the MBB. The big comment should suffice to explain this?

Reply:
I guessed it's related but only because it's part of the same PR. If I looked at the code in isolation I'm not sure I'd get the link -- "correspond" is a nothing word, really.

Reply:
OK, so it used to say:

>  // To that end, we store the indices of all corresponding BasicBlocks in the block address map.

But that was lost when you proposed the new text. Could you work it back in?

Reply:
We raced. I'll revise my comment a bit.